recipe/meta.yaml

```diff
@@ -1,16 +1,16 @@
-{% set version = "0.1.1" %}
+{% set version = "0.2" %}
 
 package:
     name: geopandas
     version: {{ version }}
 
 source:
     fn: geopandas-{{ version }}.tar.gz
-    url: https://pypi.python.org/packages/source/g/geopandas/geopandas-{{ version }}.tar.gz
-    md5: a01b5d3eb6bc097665a0ba297c39029f
+    url: https://pypi.python.org/packages/2a/1e/ea0c9ed982a62d693cbebda0da10321361a88d2a44c67292f64ca3675275/geopandas-{{ version }}.tar.gz
```

Comment:
Let's use the more elegant `https://pypi.io/packages/source/g/geopandas/geopandas-{{ version }}.tar.gz` :wink: